src/Text/Pandoc/Extensions.hs

```diff
@@ -284,14 +286,10 @@ multimarkdownExtensions = extensionsFromList
   , Ext_auto_identifiers
   , Ext_mmd_header_identifiers
   , Ext_implicit_figures
-  -- Note: MMD's syntax for superscripts and subscripts
-  -- is a bit more permissive than pandoc's, allowing
-  -- e^2 and a~1 instead of e^2^ and a~1~, so even with
-  -- these options we don't have full support for MMD
-  -- superscripts and subscripts, but there's no reason
-  -- not to include these:
-  , Ext_superscript
+  , Ext_mmd_short_subscript
```

Comment:
Thank you for your feedback! That makes sense to me. On the other hand, there are already two extensions for 'normal' sub and superscripts: Ext_superscript and Ext_supscript. Is it fine not to be consistent with those here?

Reply:
I'm ok with one extension.  But perhaps we should just call it `short_supersubscripts`?  Will it be confusing, I wonder, if enabling this without `superscript` and `subscript` will have no effect?

Reply:
It will be part of the `mmd` package of extensions, but does its name need to include `mmd`, since the rest of the name already describes it well enough?

Reply:
I feel like we should either (a) have separate `short_superscripts` and `short_subscripts` extensions or (b) have one `short_supersupscripts` extension that works to modify the behavior of `superscript` and `subscript` (and does nothing if they aren't enabled).  Otherwise there's a weird lack of parallel.

Reply:
I see what you mean. I agree we should probably pick one, but this seems like a tradeoff between configurability and complexity, and I don't think I know the pandoc project well enough to provide useful input either way. What do you suggest?

Reply:
Just to follow up on this: I'm happy to implement either, but I'm waiting on input to determine which one to go with.